nuxeo-features/nuxeo-automation/nuxeo-automation-client/src/test/java/org/nuxeo/ecm/automation/client/AutomationClientOSGiComplianceTest.java

```diff
@@ -54,6 +55,7 @@
  */
 @RunWith(PaxExam.class)
 @ExamReactorStrategy(PerMethod.class)
+@Ignore("OSGi compliance not necessary")
```

Comment:
why not removing the class?
ignore is expected to be temporary and must usually refer to a JIRA ticket